### PR TITLE
refactor editor.ts/html and mask

### PR DESCRIPTION
**Pull request by shmcaffrey**

### CODE REVIEW

File: src/main/ngapp/accio/src/app/editor/editor.component.ts [hunk 1]
```diff
@@ -15,148 +14,198 @@ import { ImageBlob } from '../ImageBlob';
...
-  url: string;
-  display: boolean = false;
-  projectId: string;
+  //  Display variables.
+  private image: HTMLImageElement;
+  imageUrl: string;
+  displayMaskForm: boolean = false;
+  disableSubmit: boolean = false;
+
+    //  Mask variables.
```
Comment (charmychancey) [inline src/main/ngapp/accio/src/app/editor/editor.component.ts:26]: nit: spacing of comment doesn't like up with the block

File: src/main/ngapp/accio/src/app/editor/editor.component.ts [hunk 2]
```diff
@@ -15,148 +14,198 @@ import { ImageBlob } from '../ImageBlob';
...
+ /** 
+  *  Sets new pixels in magenta, clears canvas of previous data 
+  *    and draws image and mask as scaled. Disables submit
+  *    on mask until the url is set. 
+  *  class @param this.disableFloodFill must equal true before called because 
+  *                               maskMmageData is being updated
+  *  class @param this.disableSubmit must equal true before called because 
+  *                            maskUrl is being updated in drawMask(). 
+  *  class @param maskPixels event Output() from mask.directive
+  *                    Gives the new pixels to add to the mask
```
Comment (charmychancey) [inline src/main/ngapp/accio/src/app/editor/editor.component.ts:162]: Nit: can you make the line spacing here consistent

Comment (shmcaffrey) [reply src/main/ngapp/accio/src/app/editor/editor.component.ts:162]: Yikes yes my bad

**Comment by shmcaffrey** [comment]
I realized why only showing the mask wasn't working - just updating that.

**Pull request merged**